plugin/trino-iceberg/src/main/java/io/trino/plugin/iceberg/IcebergConfig.java

```diff
@@ -73,6 +73,7 @@ public class IcebergConfig
     private double minimumAssignedSplitWeight = 0.05;
     private Optional<String> materializedViewsStorageSchema = Optional.empty();
     private boolean sortedWritingEnabled = true;
+    private boolean aggregationPushdownEnabled;
```

Comment:
Generally improvement that is not enabled benefits only those who know about it (those who added it + small groups of other people). If there is something  preventing the feature from being enabled, it should be code documented as a `// TODO` code comment, with a link to the issue that's focused solely on that (eg https://github.com/trinodb/trino/issues/10974 as it stands today is too broad)

Reply:
Even if this doesn't get enable by default, it should be enabled in at least one test class extending from `BaseConnectorTest`, to get reasonable test coverage.

Reply:
It should be enabled by default since it's only about `counts`, which according to spec should be accurate in metadata files

Reply:
Agreed. However, we need to understand what the approach would look like to support other aggregations than `count`, since the input information, and guarantees, is so much different.